h/feeds/rss.py

```diff
@@ -73,8 +71,7 @@ def feed_from_annotations(annotations, annotation_url, rss_url, html_url,
             for annotation in annotations]
     }
 
-    if annotations:
-        feed['pubDate'] = parser.parse(annotations[0]['updated']).strftime(
-            '%a, %d %b %Y %H:%M:%S %Z')
+    if annotations and annotations[0].updated:
```

Comment:
This is what `factories.py` is for, at some point we should refactor these tests to use that